contracts/ToucanRegenBridge.sol

```diff
@@ -68,14 +65,11 @@ contract ToucanBridge is Ownable, Pausable {
             isRegenAddress(bytes(recipient)),
             "recipient must a Regen Ledger account address"
         );
+        require(toucanContractRegistry.checkERC20(address(tco2)), "not a Toucan contract");
+
         totalTransferred += amount;
 
         emit Bridge(msg.sender, recipient, address(tco2), amount);
-
-        require(
-            nctRegistry.checkERC20(address(tco2)),
-            "contract not part of the Toucan NCT registry"
-        );
         tco2.retireFrom(msg.sender, amount);
```

Comment:
Is this currently still in the spec, or this is just a WIP for testing?

Reply:
Likely used just for testing, we need to replace with `burn` once that is properly exposed by the TCO2 contracts.